docs/getting-started/quickstart.md

```diff
@@ -61,31 +63,26 @@ Run the following command to deploy your app to DBOS Cloud:
 npx dbos-cloud app deploy
 ```
 
-This command first prompts you to login/sign up for DBOS Cloud.
-
-Then, it prompts you to provision a free [database instance](../cloud-tutorials/database-management.md) for your application.
-
-Finally, it uploads your code to DBOS Cloud and deploys your application.
-After about a minute, it should succeed and print `Successfully deployed <app-name>! Access your application at <URL>`.
+This command first prompts you to login, or register if this is your first time. Then, it prompts you to provision a free cloud database instance. Finally, it uploads your code to DBOS Cloud and deploys your app. After about a minute, it should succeed and print `Successfully deployed <app-name>! Access your application at <URL>`.
 
 To see that your app is working, visit `<URL>` in your browser. For example, if your username is `mike` and your app name is `hello`, you would visit:
 ```
 https://mike-hello.cloud.dbos.dev/
 ```
 
-Congratulations, you've successfully deployed your first application to DBOS Cloud! You can see your deployed app in the [cloud console](https://console.dbos.dev/), or in the CLI by running `npx dbos-cloud app list`.
+Congratulations, you've successfully deployed your first app to DBOS Cloud! You can see your deployed app in the [cloud console](https://console.dbos.dev/), or in the CLI by running `npx dbos-cloud app list`.
 
 
-## Run the App Locally
+## Run the App on Your Computer
 
-For development, debugging, testing, or self-hosted deployment, here's how to run this app on your local machine. This section assumes you've already created an application folder as described above.
+For development, testing, or self-hosted deployment, here's how to run this app on your local machine. This section assumes you've already created an app folder as described above.
 
 ### 1. Install Postgres or use Docker
```

Comment:
To make things even simpler, let's combine steps 1 and 2 here. Their being separate makes the doc look complicated.

Reply:
Done